capture_gui/plugins/ioplugin.py

```diff
@@ -132,32 +112,21 @@ def toggle_use_default(self):
         """Toggle if the file name and directory widgets are enabled"""
 
         state = self.use_default.isChecked()
-        self.filename_widget.setVisible(not state)
         self.dir_widget.setVisible(not state)
 
     def get_save_directory(self):
+        """Return file path in which the file will be saved"""
 
         # Maya's browser return Linux based file paths to ensure Windows is
         # supported we use normpath
-        browsed_path = os.path.normpath(lib.browse())
-        filename = browsed_path.split(os.path.sep)[-1]
-        filepath = browsed_path.split(filename)[0]
-
-        self.directory_path.setText(filepath)
-        self.filename.setText(filename)
-
-    def create_call_playblast(self, filepath):
-
-        if not os.path.isfile(filepath):
-            raise RuntimeError("Given path '{}' "
-                               "is not a file".format(filepath))
+        self.directory_path.setText(os.path.normpath(lib.browse()))
```

Comment:
Does this still error out when the browser dialog was cancelled (returning None)? As behavior I'd expect it to do nothing at all when cancelling.

```python
# psuedocode
path = lib.browse()
if path is None:
    return
self.directory_path.setText(os.path.normpath(path))
```